Content.Shared/Chat/SharedChatSystem.cs

```diff
@@ -275,7 +275,9 @@ public static string InjectTagInsideTag(ChatMessage message, string outerTag, st
     public static string InjectTagAroundString(ChatMessage message, string targetString, string tag, string? tagParameter)
     {
         var rawmsg = message.WrappedMessage;
+#pragma warning disable RA0026 // Use of uncached static Regex function
         rawmsg = Regex.Replace(rawmsg, "(?i)(" + targetString + ")(?-i)(?![^[]*])", $"[{tag}={tagParameter}]$1[/{tag}]");
+#pragma warning restore RA0026 // Use of uncached static Regex function
```

Comment:
Again, this is actually a valid warning that should have its cause fixed, not be suppressed.